src/main/java/koval/myfit/mobile/gui/pages/android/menuPages/HomePage.java

```diff
@@ -0,0 +1,57 @@
+package koval.myfit.mobile.gui.pages.android.menuPages;
+
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import koval.myfit.mobile.gui.pages.android.menu.DownMenuModal;
+import koval.myfit.mobile.gui.pages.common.loginPages.WelcomePageBase;
+import koval.myfit.mobile.gui.pages.common.menuPages.HomePageBase;
+import koval.myfit.mobile.gui.pages.service.enums.DownMenuElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = HomePageBase.class)
+public class HomePage extends HomePageBase {
+
+    @FindBy(id = "com.google.android.apps.fitness:id/bottom_navigation")
+    private DownMenuModal downMenuModal;
+
+    @FindBy(id = "com.google.android.apps.fitness:id/halo_view")
+    private ExtendedWebElement activityChartView;
+
+    @FindBy(id = "com.google.android.apps.fitness:id/og_apd_internal_image_view")
+    private ExtendedWebElement profileImage;
+
+    @FindBy(id = "com.google.android.apps.fitness:id/og_text_card_title")
+    private ExtendedWebElement signOutButton;
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public boolean isPageOpened() {
+        return activityChartView.isElementPresent(5);
+    }
+
+    @Override
+    public HomePageBase clickProfileImage() {
+        profileImage.click();
+        return initPage(getDriver(), HomePageBase.class);
+    }
+
+    @Override
+    public WelcomePageBase clickSignOutBtn() {
+        clickProfileImage();
```

Comment:
incorrect name of method clickSignOutBtn for you current implementation the better name is signOut()
you don't need use here clickProfileImage() use profileImage.click(); avoid usage method inside other method if it possible